### PR TITLE
replace latin words i.e. and e.g.

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Replace existing Latin words i.e. and e.g. according to the request of Google developer documentation style guide.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://developers.google.com/style/abbreviations#dont-use

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: certificate-authentication.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ It is recommended that you use [OpenSSL](https://www.openssl.org/) to create key
     ```bash
     Country Name (2 letter code) [AU]:US
     State or Province Name (full name) [Some-State]:California
-    Locality Name (eg, city) []:San Francisco
-    Organization Name (eg, company) [Internet Widgits Pty Ltd]:PingCAP Inc.
-    Organizational Unit Name (eg, section) []:TiDB
+    Locality Name (e.g. city) []:San Francisco
+    Organization Name (e.g. company) [Internet Widgits Pty Ltd]:PingCAP Inc.
+    Organizational Unit Name (e.g. section) []:TiDB
```
Comment (CharLotteiu) [inline certificate-authentication.md:60]: These lines are within the code block which Vale does not process, so I leave "e.g." as it is and unify the format. Same for other changes in this file.

Comment (lilin90) [reply certificate-authentication.md:60]: Got it.

**Comment by CharLotteiu** [comment]
@lilin90 PR updated. PTAL. Thanks!

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**